policies/election-procedures.md

```diff
@@ -0,0 +1,164 @@
+# Astropy Election Procedures
+
+## Overview
+
+Astropy votes are managed with the [Helios Voting platform](https://heliosvoting.org) and
+announced publicly via the astropy-dev mailing list and the OpenAstronomy Discourse forum. 
+
+## Procedure for creating an election
+
+1.  Set a date for the election and announce the election
+   - This sets various deadlines that must be followed by nominees and voting
+      members / CoCo nominees.
+   - Election start date must be at least 4 weeks after the announcement email
```

Comment:
Consistent period at the end of all sentences throughout this doc would be nice.

Reply:
Went with the no periods on bullets option.